### DISCUSSION TITLE
Transferring a GitHub Project into a Private Org

### DISCUSSION THREAD

**Discussion by Shariyar-Irfan** [community/community#19623] [Projects and Issues]
Is it possible to Transfer a GitHub Project into a Private Org?

Testing the Beta project space in a single account, it's worked very well, but now we would like to move it into our private org before it's integrated into our processes.

**Comment by Zi-Tao** [comment]
@Shariyar-Irfan https://docs.github.com/en/repositories/creating-and-managing-repositories/transferring-a-repository
